modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -79,15 +109,26 @@ and it must be loaded before the SDK.
 
 // end::install[]
 
-== Hello Couchbase
 
-Couchbase uses xref:7.0@server:learn:security/roles.adoc[Role Based Access Control (RBAC)] to control access to resources.
+== Cloud Connections
+
+If you are connecting to https://docs.couchbase.com/cloud/index.html[Couchbase Capella], be sure to get the correct endpoint as well as user, password, and `couchbasecloudbucket`
+
+For developing on Capella, if you are not working from the same _Availability Zone_, refer to the following:
+
+* Notes on xref:ref:client-settings.adoc#constrained-network-environments[Constrained Network Environments],
+* xref:project-docs:compatibility.adoc#network-requirements[Network Requirements],
+* If you have a consumer-grade router which has problems with DNS-SRV records review our xref:howtos:troubleshooting-cloud-connections.adoc#troubleshooting-host-not-found[Troubleshooting Guide].
+
+=== Preamble XXXX
```

Comment:
Australian lager?

Reply:
Yeah... I have some more work to do here...